firefox-ios/Client/Frontend/Fakespot/Views/FakespotMessageCardView.swift

```diff
@@ -194,25 +196,45 @@ final class FakespotMessageCardView: UIView, ThemeApplicable, Notifiable {
         fatalError("init(coder:) has not been implemented")
     }
 
-    func configure(_ viewModel: FakespotMessageCardViewModel) {
+    func configure(
+        _ viewModel: FakespotMessageCardViewModel,
+        fakespotViewModel: FakespotViewModel? = nil
```

Comment:
I'm not really comfortable with the idea of passing 2 viewModels. I think we should have a way to update the view if the progress change.